packages/istanbul-reports/test/lcovonly/index.js

```diff
@@ -26,23 +26,30 @@ describe('LcovOnlyReport', () => {
             __dirname,
             '../fixtures/specs/' + file
         ));
-        it(fixture.title, function() {
-            if (isWindows()) {
-                // appveyor does not render console color.
-                return this.skip();
-            }
-            const context = istanbulLibReport.createContext({
-                dir: './',
-                coverageMap: istanbulLibCoverage.createCoverageMap(fixture.map)
-            });
-            const tree = context.getTree('pkg');
-            const report = new LcovOnlyReport(fixture.opts);
-            tree.visit(report, context);
-            const output = FileWriter.getOutput().replace(/SF:.*/, 'SF:');
-            if (fixture.lcovonlyExpected) {
+        if (fixture.lcovonlyExpected) {
```

Comment:
This seems a little strange, why are we only running these next lines for the one specific fixture?